frontend/appflowy_flutter/integration_test/import_page_from_notion_test.dart

```diff
@@ -0,0 +1,166 @@
+import 'dart:io';
+
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import 'util/mock/mock_file_picker.dart';
+import 'util/util.dart';
+import 'package:path/path.dart' as p;
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+  group('import file from notion', () {
+    testWidgets('import markdown zip from notion', (tester) async {
+      const mainPageName = 'AppFlowy Test';
+      const subPageOneName = 'Appflowy Subpage 1';
+      const subPageTwoName = 'AppFlowy Subpage 2';
+      const subSubPageName = 'AppFlowy SubSub Page';
+      final context = await tester.initializeAppFlowy();
+      await tester.tapGoButton();
+
+      // expect to see a readme page
+      tester.expectToSeePageName(gettingStarted);
+
+      await tester.tapAddViewButton();
+      await tester.tapImportButton();
+
+      final paths = <String>[];
+      final ByteData data = await rootBundle
+          .load('assets/test/workspaces/import_page_from_notion_test.zip');
+      final path = p.join(
+        context.applicationDataDirectory,
+        'import_page_from_notion_test.zip',
+      );
+      paths.add(path);
+      final file = File(path);
+      await file.writeAsBytes(
+        data.buffer.asUint8List(data.offsetInBytes, data.lengthInBytes),
+      );
+      // mock get files
+
+      expect(find.widgetWithText(Card, 'From Markdown Zip'), findsNothing);
+      await tester.tapButtonWithName('Import from Notion Markdown Zip');
+      expect(find.widgetWithText(Card, 'From Markdown Zip'), findsOneWidget);
+      await tester.tapButtonWithName('From Markdown Zip');
+      expect(find.text('Import Notion From Markdown Zip'), findsOneWidget);
+      await mockPickFilePaths(
+        paths: paths,
+      );
+      await tester.tapButtonWithName('Upload zip file');
+      tester.expectToSeePageName(mainPageName);
+      await tester.openPage(mainPageName);
```

Comment:
The import panel should close automatically after selecting files.

Also, avoid using `openPage` to close the panel. Even if you intend to close it, you can simply tap on a blank area, like offset(0, 0).